app/models/async_transaction/va_profile/base.rb

```diff
@@ -43,8 +43,12 @@ def self.last_requested_for_user(user)
       #
       def self.start(user, response)
         # vet360_id is no longer required for Contact Information API V2
-        source_id = Flipper.enabled?(:va_v3_contact_information_service, user) ? user.vet360_id || 0 : user.vet360_id
```

Comment:
Update source_id if using new service that does not require user to have a vet360_id